packages/airnode-node/src/evm/requests/blocking.ts

```diff
@@ -61,7 +60,7 @@ export function applySponsorAndSponsorWalletRequestLimit([
     }
     const sponsorRequests = requestCountPerSponsorId.get(sponsorId)!;
 
-    if (sponsorRequests < MAXIMUM_SPONSOR_WALLET_REQUESTS) {
+    if (sponsorRequests < 5) {
```

Comment:
Ah- what we're looking for here is not to replace the constant with a hardcoded value, but rather to remove the conditional completely i.e. so that there is no hardcoded limit on the number of sponsor requests. This will also require the tests be updated.